paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetViewModelTest.kt

```diff
@@ -451,8 +451,29 @@ class CustomerSheetViewModelTest {
             assertThat(awaitItem())
                 .isInstanceOf(SelectPaymentMethod::class.java)
             viewModel.handleViewAction(CustomerSheetViewAction.OnAddCardPressed)
+
+            val item = awaitItem()
+            assertThat(item).isInstanceOf(AddPaymentMethod::class.java)
+            assertThat((item as AddPaymentMethod).formViewData.elements).isNotEmpty()
```

Comment:
Should we check that all card elements are being shown here?

Reply:
Wouldn't hurt. Fine to do as a follow up if it's not easy.